bower.json

```diff
@@ -29,6 +29,7 @@
     "core-doc-viewer": "Polymer/core-doc-viewer#0.8-preview",
     "core-icon": "Polymer/core-icon#0.8-preview",
     "core-icons": "Polymer/core-icons#0.8-preview",
+    "test-fixture": "Polymer/test-fixture#master",
```

Comment:
My lack of bower-fu is showing but I don't really understand how we should deal with components depending on different versions? For example, if this element has `Polymer/test-fixture#^0.0.2` and somebody else has `Polymer/test-fixture#master`, is that going to cause problems?